### ISSUE TITLE
Document purpose for tf.keras.preprocessing module

### ISSUE THREAD

**Issue by mjul** [tensorflow/tensorflow#37445]
## URL(s) with the issue:

https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing

## Description of issue (what needs changing):

The module documentation is very terse and only states that it contains "preprocessing utils". It does not state the specific purpose.

It would be helpful if it defined the intended use of the `tf.keras.preprocessing` module (_e.g._ to clean up or transform tf.data.Datasets before they are fed to the model).

Also, since the `tf.feature_column` module has similar functionality, it would be nice to describe when to use one or the other, or how they are intended to be used together.

**Comment by ManishAradwad** [comment]
I'll start working on it...
Thanks!

**Comment by jvishnuvardhan** [comment]
@ManishAradwad Can you please link the PR with this issue? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
